enrollment-server-onboarding-provider-innovatrics/src/main/java/com/wultra/app/onboardingserver/provider/innovatrics/InnovatricsConfigProps.java

```diff
@@ -60,4 +60,14 @@ class InnovatricsConfigProps {
      */
     private RestClientConfiguration restClientConfig;
 
+    private PresenceCheckConfiguration presenceCheckConfiguration;
+
+    @Getter @Setter
+    public static class PresenceCheckConfiguration {
+        /**
+         * Presence check minimal score threshold.
+         */
+        private double score;
```

Comment:
The implicit default value here is `0`, is this correct?

Reply:
It should be taken from application properties, but yes, I may be explicit here to be safe.